### PR TITLE
ORCA-724: Update Operator documentation for recovery

### PR DESCRIPTION
**Pull request by zroehrich**
## Summary of Changes

Addresses [ORCA-724: Update Operator documentation for recovery](https://bugs.earthdata.nasa.gov/browse/ORCA-724)

## Changes

* Updated documentation to update inputs and outputs in  `website/docs/operator/data-recovery.md`.

## Type of change

Please delete options that are not relevant.
- [x] This change requires a documentation update

## How Has This Been Tested?
Deployed website locally and verified that changes worked successfully.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
    - [x] Operator Documentation 
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Manual tests (outlined above)

### CODE REVIEW

**Review by rizbihassan — CHANGES_REQUESTED** [review]
The process of recovery has changed after ORCA v8.1. We should more information to the webpage relevant to the below two points.


1) We no longer use [CMA](https://github.com/nasa/cumulus-message-adapter).
2) Users now need to deploy a recovery adapter first before they can run recovery. See https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus#modify-the-recovery-workflow

File: website/docs/operator/data-recovery.md [hunk 1]
```diff
@@ -111,10 +110,6 @@ The following is an input event example that an operator might set up while runn
       ]
     }
   },
-  "cumulus_meta": {
-    "system_bucket": "test-internal",
-    "asyncOperationId": "1234"
-  }
```
Comment (rizbihassan) [inline website/docs/operator/data-recovery.md:117]: Any reason why this proprty was removed?

Comment (zroehrich) [reply website/docs/operator/data-recovery.md]: Reverted this change

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ and includes an additional section for migration notes.
 - *ORCA-361* Removed hardcoded test values from `extract_file_paths_for_granule` unit tests.
 - *ORCA-710* Removing duplicate logging messages in `integration_test/workflow_tests/custom_logger.py`
 - *ORCA-784* Changed documentation to replace restore with copy based on task's naming as well as changed file name from `website/docs/operator/restore-to-orca.mdx` to `website/docs/operator/reingest-to-orca.mdx`.
+- *ORCA-724* Updated documentation to update inputs and outputs in  `website/docs/operator/data-recovery.md`.
```
[the remark this answers is not in the document]
Comment (zroehrich) [reply CHANGELOG.md]: Reworded changelog and added in an info block about no longer using CMA and needing to deploy the recover adapter before running a recovery

**Review by rizbihassan — APPROVED** [review]

**Pull request merged**